<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FDumperv2Class</class>
 <widget class="QMainWindow" name="FDumperv2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FDumperv2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="ImageListtxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>306</width>
      <height>176</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="galleryTabs">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>20</y>
      <width>331</width>
      <height>476</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QListView" name="queueList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>365</y>
      <width>306</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>315</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Download Queue</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>306</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="queueLineInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="addToQueue">
        <property name="text">
         <string>Queue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>210</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Content Filtering</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>321</width>
      <height>206</height>
     </rect>
    </property>
    <property name="title">
     <string>Image List</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>341</width>
      <height>506</height>
     </rect>
    </property>
    <property name="title">
     <string>Preview Gallery</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>306</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="ctfilterMain">
       <property name="text">
        <string>Main Gallery</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="ctfilterScraps">
       <property name="text">
        <string>Scraps</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="ctfilterFavs">
       <property name="text">
        <string>Favorites</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="allRatings">
       <property name="text">
        <string>All Ratings</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="SFWonly">
       <property name="text">
        <string>SFW Only</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="NSFWonly">
       <property name="text">
        <string>NSFW Only</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>ctfilterMain</zorder>
    <zorder>ctfilterScraps</zorder>
    <zorder>allRatings</zorder>
    <zorder>SFWonly</zorder>
    <zorder>NSFWonly</zorder>
    <zorder>ctfilterFavs</zorder>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>queueList</zorder>
   <zorder>ImageListtxt</zorder>
   <zorder>galleryTabs</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FDumperv2.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
